src/com/javacore/lesson10/HomeWork10.java

```diff
@@ -0,0 +1,22 @@
+package com.javacore.lesson10;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HomeWork10 {
+
+    public static void main(String[] args) throws InterruptedException {
+
+        List<Thread> threads = new ArrayList<>();
+        CounterThread counterThread = new CounterThread(10, 1000);
+
+        System.out.println("Counter before starting threads: " + counterThread.getCounter());
+
+        counterThread.startThreads(counterThread, threads, true);
```

Comment:
will be better to return List<Thread> threads from startThreads
It's not correct to pass empty list to fill it in the void method, it's confusing)

Reply:
Done